.github/workflows/build-docker.yml

```diff
@@ -37,6 +37,7 @@ jobs:
 
       - name: Save image artifact
         uses: actions/upload-artifact@v3
+        if: github.event_name != 'release'
```

Comment:
This means we can't download the test builds to check. At minimum, manual runs need to upload

Reply:
We'll still get artifacts for non-release builds, and release builds will instead have their artifacts uploaded to the release itself.